CONTRIBUTING.md

```diff
@@ -148,14 +148,30 @@ During a peer review, reviewers will add a tag to the comment to help the author
 [Kudos]|The reviewer liked this, specifically. Go you.
 [Suggestion]| An optional, alternative approach.
 
-
-
 Adding multiple tags to a comment is fine.  Do not combine tags.
 
 Good: ``[Suggestion][Kudos] This is great, but...``
 
 Bad:  ``[Suggestion|Kudos] This is great, but...``
 
+
+## Pull request labels
+Labels may be applied to your pull requests, for the purposes of organization and / or overall 
+status.
+
+| label | Explanation |
+| --- | --- |
+Awaiting changes    | this pull request is on hold until requested changes have been effected
+Feature             | this pull request adds a new feature
+Bug Fix             | this pull request fixes some bug
+Refactor            | this pull request refactors something
+Documentation       | this pull request only effects documentation
+Linting             | this pull request adjusts our linting configurations
+CI change           | This pull request reconfigures the CI environment
+Dependency change   | this pull request modifies our dependencies
```

Comment:
[Error] Please keep perspective. As much as it is 'our' project, properly written it should be *the* project :)

``Dependency Change | Modifies project dependencies``
``Policy | Change to contributing expectations``

These are examples, but I think you the idea.

Reply:
I argue this is not an error but style. 

At any rate, will effect

Reply:
resolved in 0c964a9